lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person_hash = {favorite_number: 420, first_name: Sampson}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'new_hash = Hash.new { |hash, name| hash[name]= "Dr said #{name} needs a backiatomy." }'
```

Comment:
This is using Hash.new, we wanted the shorthand syntax:

`'person.default = "#{person[:first_name]} is great"'`